quickstarts/host-inventory-quickstart.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: console.openshift.io/v1
+kind: ConsoleQuickStart
+metadata:
+  name: host-inventory
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    capability.openshift.io/name: Console
+spec:
+  displayName: Get started with on-premise host inventory
+  durationMinutes: 20
+  description: Configure your host inventory and create a cluster.
+  prerequisites: 
+    - A storage operator is required (e.g. OpenShift Data Foundation) in order for the assisted service to run correctly. To install the OpenShift Data Foundation, complete the "Install OpenShift Data Foundation" quick start.
```

Comment:
I mentioned this earlier but didn't look it up....let me check on eg usage.

Reply:
This is style manual guidance for e.g.:


Do not use e, e-   adjective
Do not create terms using the prefix "e" or "e-".

Reply:
CSS | Stands for Cascading Style Sheets. Write CSS in all caps.
-- | --
e.g., i.e., and etc. | Use sparingly. These terms aren’t easily understood by everyone, especially users whose native language is not rooted in Latin. Write out their meaning instead:e.g. – for examplei.e. – in other wordsetc. – and more/so on

IBM style is the parent, says no. PatternFly also warns against it...


Let's instead write:

**Example:**

Reply:
Thanks @swopebe - changes are made